src/server.py

```diff
@@ -468,11 +479,55 @@ def run_seta_details_query():
     for d in cursor.fetchall():
         parts = d[0].split("'")
         jobtype.append([parts[1], parts[3], parts[5]])
-    retVal[date] = jobtype
 
+    if active:
+        alljobs = jobtype_query()
+        active_jobs = []
+        for job in alljobs:
+            found = False
+            for j in jobtype:
```

Comment:
jobtype is the array of jobtypes from the query (jobs we can run optionally), alljobs is the master list of all jobtypes, so here we are building a list of active_jobs which are alljobs-jobtype